src/agoradatatools/etl/transform/gene_info.py

```diff
@@ -101,7 +101,7 @@ def transform_gene_info(
     resource_url_suffix = "%22%5D%7D%5D%7D"
     tep_info["resource_url"] = tep_info.apply(
         lambda row: resource_url_prefix + row["hgnc_symbol"] + resource_url_suffix
-        if (row["is_adi"] == True) | (row["is_tep"] == True)
```

Comment:
Just want to draw some attention to this change. I don't think it changes the behavior of this function.